action.yml

```diff
@@ -9,16 +9,27 @@ inputs:
   cosign-release:
     description: 'Cosign release version to use in the actions.'
     required: false
-    default: 'v0.2.0'
+    default: 'v0.4.0'
 runs:
   using: "composite"
   steps:
-    - run: curl -L https://storage.googleapis.com/cosign-releases/v0.2.0/cosign-linux-amd64 -o cosign
+    - run: |
+        sudo apt-get update -q
+        sudo apt-get install -yq \
+             libpcsclite1
```

Comment:
I can't see this used anywhere below. Why do we need this installed?

Reply:
it's used by cosign itself (for yubikey support). We do a similar step in `sigstore/cosign`: https://github.com/sigstore/cosign/blob/main/.github/workflows/build.yaml#L39-L40